360i.html

```diff
@@ -0,0 +1,111 @@
+<html>
+<head>
+	
+	<h1>360 i</h1>
+<h2>
+<div id="box">
+	<div class = "box" id = "LogIn"><a href="main.html">LOG IN</a></div></h2>
```

Comment:
I think we talked about this in class, but I would definitely eliminate the <h2> tag, or at the very least, only wrap it around the Log In text: `<div class = "box" id = "LogIn"><a href="main.html"><h2>LOG IN</h2></a></div>`